### PR TITLE
6213 Спиридонова К.С Lab5-var6

### PR DESCRIPTION
**Pull request by kksseonn**

### CODE REVIEW

**Review by alxmcs — CHANGES_REQUESTED** [review]
image
А куда train accuracy делся?

И было бы неплохо вписать в ноутбук какие-нибудь выводы на основе проведенных экспериментов и оценить работу итоговой модели

**Review by alxmcs — CHANGES_REQUESTED** [review]
Может я куда-то не туда смотрю, но  все еще не вижу на графике training accuracy, только одинокие validation accuracy

**Comment by kksseonn** [comment]
> Может я куда-то не туда смотрю, но все еще не вижу на графике training accuracy, только одинокие validation accuracy

еще не переделала ноутбук :(

**Review by alxmcs — COMMENTED** [review]
А, прошу прощения.
Был нотификейшен о коммите в pr, запроса повторного ревью не было.
Зря быканул.

**Review by alxmcs — APPROVED** [review]
Зачтено, однако:

>Слишком высокий learning rate (0.1) приводит к расходиму модели.
